src/components/dashjs-api-control/dashjs-api-control.tsx

```diff
@@ -144,6 +162,78 @@ export class DashjsApiControl {
     toast.present();
   }
 
+  updateInformation(action : number, event : any) {
```

Comment:
How about `updateLocalStorageInformation`?

Reply:
Maybe also do it in two different functions, because the `action` is not really speaking for itself and the function is doing two different things?

Reply:
Yea, it stores either autostart boolean or the mediaurl string.. the difference isn't big so I wanted to add them both into one function otherwise there will be two functions that look almost identical

Reply:
But these are two completely different things? :D Like I could see it if something would be reused but it's two different things from the get go